WordPress/src/androidTest/java/org/wordpress/android/e2e/LoginTests.kt

```diff
@@ -59,9 +59,4 @@ class LoginTests : BaseTest() {
             )
             .confirmLogin(true)
     }
-
-    @After
-    fun tearDown() {
-        logoutIfNecessary()
-    }
```

Comment:
Every `setUp` method (for all tests) contains `logoutIfNecessary`, so I saw no value in also having a teardown that does the same.

Reply:
usually the "clean up" happens after the test ends, curious to know your reason for keeping the one in `setUp()` instead of `tearDown()`?

Reply:
I'd advocate that every test is responsible for setting up the right environment before it starts, and not when it ends. Thus from my experience, it's always more resilient to handle prerequisites in setUp.

Reply:
Only `LoginTests.kt` and `SignUpTest.kt` contained both `setUp` and `teardDown` with `logoutIfNecessary`, while the remaining five files contain only `setUp` with `logoutIfNecessary`. I removed the ones from `tearDown` because of a smaller footprint.

Reply:
i've personally used both `setUp()` and `tearDown()` to "clean up" test states, depending on what it does and what the end states of the tests are. in the case where we are deleting the app (what JPiOS is doing), it would make sense to do it during `tearDown()` and from your reasoning for this one, this change makes sense to me since it was already in `setUp()` in the other files 👍